domains.txt

```diff
@@ -362,15 +403,19 @@ jitr.co.uk
 jiyuconsulting.co.uk
 jktech.com
 jmdigital.com
+job-matchers.co.uk
+jobnjob.co.uk
 jobsinscience.com
 jobstheword.co.uk
 jobstheword.com
 jp-engineering.co.uk
 jp-jobs.com
+jpeople.co.uk
```

Comment:
JPeople - Our aim is not to be the biggest, but to be the best supplier of skilled Java and Open Source professionals in Europe